Jenkinsfile

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env groovy
```

Comment:
I add this out of habit to ensure the editor knows how to syntax highlight the document.